src/interface/orthnull.jl

```diff
@@ -19,43 +19,47 @@ end
 # Orth functions
 # --------------
 """
-    left_orth(A; [kind::Symbol, atol::Real=0, rtol::Real=0, alg]) -> V, C
-    left_orth!(A, [VC]; [kind::Symbol, atol::Real=0, rtol::Real=0, alg]) -> V, C
+    left_orth(A; [kind::Symbol, trunc, alg_qr, alg_polar, alg_svd]) -> V, C
+    left_orth!(A, [VC]; [kind::Symbol, trunc, alg_qr, alg_polar, alg_svd]) -> V, C
 
 Compute an orthonormal basis `V` for the image of the matrix `A` of size `(m, n)`,
 as well as a  matrix `C` (the corestriction) such that `A` factors as `A = V * C`.
 The keyword argument `kind` can be used to specify the specific orthogonal decomposition
-that should be used to factor `A`, whereas `atol` and `rtol` can be used to control the
+that should be used to factor `A`, whereas `trunc` can be used to control the
 precision in determining the rank of `A` via its singular values.
 
+`trunc` can either be a truncation strategy object or a NamedTuple with fields
+`atol`, `rtol`, and `maxrank`.
+
 This is a high-level wrapper and will use one of the decompositions
-`qr!`, `svd!`, and `left_polar!` to compute the orthogonal basis `V`, as controlled
+`qr_compact!`, `svd_compact!`/`svd_trunc!`, and `left_polar!` to compute the orthogonal basis `V`, as controlled
```

Comment:
```suggestion
[`qr_compact!`](@ref), [`svd_compact!`](@ref)/[`svd_trunc!`](@ref), and [`left_polar!`](@ref) to compute the orthogonal basis `V`, as controlled
```
(plus maybe reformatting)